src/ontology/ro-edit.owl

```diff
@@ -22,17 +22,18 @@ Import(<http://purl.obolibrary.org/obo/ro/components/el-constraints.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/components/generated-axioms.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/components/rohom.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/components/temporal-intervals.owl>)
-Import(<http://purl.obolibrary.org/obo/ro/imports/caro_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/cl_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/envo_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/go_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/omo_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/orcidio_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/other_import.owl>)
 Import(<http://purl.obolibrary.org/obo/ro/imports/pato_import.owl>)
+Import(<http://purl.obolibrary.org/obo/ro/releases/2023-05-24/imports/uberon_import.owl>)
```

Comment:
I saw that but didn't really understand how to avoid it - I did the import linkage via protege, which I assumed was reading the name of the import file, which was controlled by odk creation ?  Won't this change get overwritten the next time uberon_import.owl gets written?

Reply:
Or are we relying on the persistence of catalog-v001.xml ?  I thought that was a more "temperamental" file that could be deleted and recreated at will, each time protege was opened for example on ro-edit.owl

Reply:
You can control the flakiness using the "auto-update: False" property in the group tag of the catalog file!

Reply:
You can control the flakiness using the "auto-update: False" property in the group tag of the catalog file!

Reply:
It's better to do the linkage with a text editor;